regress/Makefile.in

```diff
@@ -48,6 +48,7 @@ check-unit:
 include core/tests.mk
 include loader/tests.mk
 include dumper/tests.mk
+include real/tests.mk
```

Comment:
Pretty sure this was recently stripped out of main

Reply:
yes, I removed it in https://github.com/postgis/postgis/pull/638